x-pack/plugins/security_solution/public/management/cypress/e2e/response_actions/response_console_mocked_data.cy.ts

```diff
@@ -24,12 +24,12 @@ import {
 } from '../../tasks/isolate';
 import { login } from '../../tasks/login';
 
-describe('Response console', { tags: '@ess' }, () => {
+describe('Response console', { tags: ['@ess', '@serverless'] }, () => {
   beforeEach(() => {
     login();
   });
 
-  describe('`isolate` command', () => {
+  describe('`isolate` command', { tags: ['@brokenInServerless'] }, () => {
```

Comment:
I did not know we could do this. So `tags` work on child `describe()` blocks? and not just the top-level `describe()`?

Reply:
correct